thirdparty/bson-cpp/CMakeLists.txt

```diff
@@ -29,6 +29,8 @@ add_library(bsoncpp
         "util/hex.h"
         "util/misc.h"
         "util/optime.h"
+        "system_info.h"
```

Comment:
Haven't reviewed the complete code yet. One high level comment - we NEVER change any code in `thirdparty` directory for licensing reasons. It seems like the changes you made don't have to be in `bson-cpp`.

Reply:
Sure @apkar, we shall move them out of thirdparty directory and raise a commit